_posts/2020-04-03-rendering-in-UE4.md

```diff
@@ -0,0 +1,32 @@
+---
+author: "alepaoletti"
+layout: post
+did: "blog10"
+title: "Rendering in Unreal Engine 4"
+slug: "RenderingInUE4"
+date: 2020-04-06 08:00:00
+categories: rendering
```

Comment:
Welcome to the magic world of this blog.

The "categories" field, in theory, should be free, but in practice, we are constrained to a fixed set of categories defined in https://github.com/zurutech/zurutech.github.io/blob/master/_config.yml#L46.

As you can see, we have a different color for every different "category", therefore you have two options:

1. Add a new category "rendering" in the _configyml file
2. Change the category to the most similar among the one available, e.g. "graphics"

We need this, otherwise, the js code in the home page fails to execute :(

Reply:
I thought that was just for _tags_, I'll change the category to _graphics_ then!